tokio/src/executor/current_thread/scheduler.rs

```diff
@@ -1,5 +1,5 @@
-use super::{Borrow, BorrowSpawner};
-use crate::park::Unpark;
+use crate::executor::current_thread::{Borrow, BorrowSpawner};
```

Comment:
Why change this away from `super`?

Reply:
I like being able to see the full path when reading code. I find it easier than doing the relative calculation based off of where I am at.

Also, always using absolute paths removes the decision of having to pick relative vs. crate absolute. Less decisions is better.